README.md

```diff
@@ -250,20 +194,14 @@ Advanced topics are available in [the official docs](https://www.apollographql.c
 * [subscriptions.md](https://www.apollographql.com/docs/android/advanced/subscriptions/)
 * [migrations.md](https://www.apollographql.com/docs/android/essentials/migration/)
 
-
-## Changelog
```

Comment:
I think this is nice since it gets picked up and be available in getting started page on the docs website

Reply:
It's just above, in the `Release` section.

Reply:
Nope, I just didn't see it. I think this is all good.